lib/trento/application/integration/discovery/discarded_event.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Trento.Integration.Discovery.DiscardedEvent do
+  @moduledoc """
+  This module contains the schema used to store an append log of the discarded events,
+  for debugging and auditing purposes.
+  No changeset is defined here, since the schema is used to store append-only data.
+  """
+
+  use Ecto.Schema
+
+  @type t :: %__MODULE__{}
+
+  schema "discarded_events" do
```

Comment:
I would rename this `discovery_events_dead_letters`.
it is a bit of a nitpick, but the pattern is broadly known, and also "discarded_events" sounds a bit generic because it is not clear to which events we are referring. For instance, we also have domain events, and in the future, we could add even more integration events.

Reply:
Done: https://github.com/trento-project/web/pull/542/commits/e7a279a8027c9b5229f62f8371f6d5d6e4f645a4